perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1858,7 +1861,10 @@ sub check_for_category : Private {
     # Just check to see if the filter had an option
     $category ||= $c->get_param('filter_category') || '';
     $c->stash->{category} = $category;
-
+    if ((my $description = $c->get_param('filter_description')) && $c->cobrand->moniker eq 'zurich' && $category) {
```

Comment:
Sorry, I only just thought of this when coming to this as a whole, but I think this should probably be called 'prefill_description', not 'filter_description' - the category one is called filter_category because it is the variable to filter by category on the `/around` page; this one isn't filtering at all :)
And that led me on to the other annoying thought - should we actually add a new prefill_category option rather than rely on category/filter_category? I say that because currently you have to supply a description for this to kick in, but going back to their quote they don't say that they'll always be supplying one, they might want to only supply a category prefill - and then we can't really use category/filter_category because then if there's normal use of those the category options would disappear.
So, hopefully nothing big, but if we change this if to `if ($c->cobrand->moniker eq 'zurich' && ((my $description = $c->get_param('prefill_description')) || (my $prefill_category = $c->get_param('prefill_category')))) {` and change next couple of lines/test accordingly to cope, I think that's all, no need to change template.

Reply:
Quite right!